### DISCUSSION TITLE
如何设置官网代码显示块的样式

### DISCUSSION THREAD

**Discussion by lin-mt** [DouyinFE/semi-design#203] [Q&A] [answered]
因为我是一个后端开发，目前也有用到 monaco ，但是调样式对我来说太难了😭，官网的样式很漂亮啊😄，能否给个样式设置呢，或者到哪可以找到相关的代码呢（我实在找不到了～～～），我只需要做简单的代码显示即可～！

万分感谢🙏！

![image](https://user-images.githubusercontent.com/58337359/140464961-065cbf64-a661-4a0b-8cfd-f5f1a24c0fa5.png)

**Comment by shijiatongxue** [comment]
@DaiQiangReal can help？

**Comment by SudoUserReal** [comment, marked as answer]
https://github.com/brijeshb42/monaco-themes    亮色 `Xcode_default.json`  暗色 `Night Owl` @lin-mt

**Reply by lin-mt** [reply]
谢谢～！
